### PR TITLE
Add build info page

### PR DESCRIPTION
**Pull request by dkurepa**

### CODE REVIEW

File: src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor [hunk 1]
```diff
@@ -1,29 +1,108 @@
...
+            <h4>
+                <span> @GetRepository() </span>
+                <span> @_build.AzureDevOpsBuildNumber </span>
+                <FluentButton IconEnd="@(new Icons.Regular.Size16.Copy())" Appearance="Appearance.Accent" OnClick="@(() => SetBarClipboard(_build.Id.ToString()))">
+                    @_build.Id
+                </FluentButton>
+            </h4>
+        </div>
+        <!-- TODO this should check if there's a failed or ongoing build that's supposed to go to this channel and display the info here -->
+        <FluentButton Appearance="Appearance.Accent" IconEnd="@(new Icons.Regular.Size16.Open())" OnClick="@(() => RedirectToUri(GetBuildUri()))">
```
Comment (premun) [inline src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: Why not just use `Href` and `Target="_blank"`? We would avoid all of this "open new window" stuff

File: src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor [hunk 2]
```diff
@@ -1,29 +1,108 @@
...
-<h3>Build</h3>
-@if (_build == null)
+@if (_build == null || _commit == null)
 {
     <p><em>Loading...</em></p>
 }
 else
 {
-    <h1>@_build.AzureDevOpsRepository</h1>
+    <div class="opposite">
```
Comment (premun) [inline src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:16]: Do we need to define our own classes already? Can't we just go with what is available? The layout section of the help page has quite a few of these flex items

File: src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor (hunk 1 above)
Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: I couldn't find a FluentUI component that would do this. Maybe we could style an anchor to look like the button?

Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: maybe this `FluentNavLink`, I'll give it a try

Comment (premun) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: https://www.fluentui-blazor.net/Anchor

Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: it's not that great to be honest, I had the best results with the js that's in this PR

Comment (premun) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: What was not working with the Anchor?

Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: The anchor does work, but we'd need to style it to look consistent with Fluent UI

Comment (dkurepa) [reply src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor:27]: Ah sorry I didn't see what you linked

**Pull request closed without merging**